docs/source/users.rst

```diff
@@ -0,0 +1,163 @@
+Users
+=====
+
+Available users in a Plone site can be created, queried, updated and deleted by interacting with the ``/@users`` endpoint on portal root (requires an authenticated user):
+
+List Users
+----------
+
+To retrieve a list of all current users in the portal, call the ``/@users`` endpoint with a GET request:
+
+.. example-code::
+
+  .. code-block:: http-request
+
+    GET /@users HTTP/1.1
+    Host: localhost:8080
+    Accept: application/json
+
+  .. code-block:: curl
+
+    curl -i -H "Accept: application/json" --user admin:admin -X GET http://localhost:8080/Plone/@users
+
+  .. code-block:: httpie
+
+    http -a admin:admin GET http://localhost:8080/Plone/@users Accept:application/json
+
+  .. code-block:: python-requests
+
+    requests.post('http://localhost:8080/Plone/@users', auth=('admin', 'admin'), headers={'Accept': 'application/json'})
+
+The server will respond with a list of all users in the portal:
+
+.. literalinclude:: _json/users.json
+   :language: js
+
+
+Create User
+-----------
+
+To create a new user, send a POST request to the global ``/@users`` endpoint with a JSON representation of the user you want to create in the body:
+
+.. example-code::
+
+  .. code-block:: http-request
+
+    POST /@users HTTP/1.1
+    Host: localhost:8080
+    Accept: application/json
+    Content-Type: application/json
+
+    {
+        'username': 'noam',
+        'email': 'noam.chomsky@mit.edu',
+        'password': 'colorlessgreenideas',
+        'username': 'noamchomsky',
+        'fullname': 'Noam Avram Chomsky',
```

Comment:
@lukasgraf Yes. That was a conscious design decision. A REST API should not expose implementation details of the underlying system IMO. I'm not sure why plone.api did not abstact that away already.

I guess PAS is storing those params separately because you might want to store the credentials separated from the user properties sometimes (e.g. if you want "users" as objects). I'd consider that an implementation detail that should not be exposed to the user of an API. 

@sneridagh @jensens opinions?

@zupo do recall why for plone.api it was decided to pass the "properties" param when creating a user instead of providing all user properties as a flat structure (named params)?